src/wmnds/components/form-elements/radios/_properties.njk.json

```diff
@@ -0,0 +1,115 @@
+{
+  "wmndsRadiosProps": [
+    {
+      "name": "items",
+      "type": "array",
+      "description": "<strong>Required.</strong> Array of radio items objects. See <code class='wmnds-website-inline-code'>items</code> below.",
+      "arrayOptions": [
+        {
+          "name": "contentText",
+          "type": "string",
+          "description": "<strong>Required.</strong> If <code class='wmnds-website-inline-code'>contentHTML</code> is set, this is not required. Text to use within each radio item label. If <code class='wmnds-website-inline-code'>contentHTML</code> is provided, the <code class='wmnds-website-inline-code'>contentText</code> argument will be ignored."
+        },
+        {
+          "name": "contentHTML",
+          "type": "string",
+          "description": "<strong>Required.</strong> If <code class='wmnds-website-inline-code'>contentText</code> is set, this is not required. HTML to use within each radio item label. If <code class='wmnds-website-inline-code'>contentHTML</code> is provided, the <code class='wmnds-website-inline-code'>contentText</code> argument will be ignored."
+        },
+        {
+          "name": "checked",
+          "type": "boolean",
+          "description": "If true, radio will be checked."
+        },
+        {
+          "name": "name",
+          "type": "string",
+          "description": "Specific name for the radio item. If omitted, then component global name string will be applied."
+        },
+        {
+          "name": "disabled",
+          "type": "boolean",
+          "description": "If true, radio will be disabled."
+        }
```

Comment:
There are some properties missing which are in the example. See the example review comments for more.